packages/bpk-component-rating/src/BpkRating.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import clamp from 'lodash.clamp';
+import { cssModules } from 'bpk-react-utils';
+import BpkText from 'bpk-component-text';
+
+import STYLES from './BpkRating.scss';
+// import RATING_SIZES from './BpkRatingSizes'; TODO: make use of this
+
+const getClassName = cssModules(STYLES);
+const MEDIUM_RATING_THRESHOLD = 6;
```

Comment:
I think these should be configurable... Good to have sensible defaults, but ultimately the consumer should be able to specify their own ranges IMO.

Reply:
So for these they will never changed as confirmed by design. Whenever used they need to be consistent so letting user land set these will cause them to be inconsistent